packages/web/src/common/store/pages/history/lineups/sagas.ts

```diff
@@ -54,7 +54,7 @@ function* getHistoryTracks() {
         ? processedTracksMap[decodeHashId(activity.item.id)!]
         : null
       // Prevent history for invalid tracks from getting into the lineup.
-      if (trackMetadata) {
+      if (trackMetadata && !trackMetadata.is_unlisted) {
```

Comment:
didn't feel worth doing in DN (additional slightly involved sql queries) so opted for this simple client change instead. lmk if strong opinions.

Reply:
This is probably fine for now.. but I think we'll want to do it at the API level to be consistent and have consistent 3rd party experiences too